packages/core/contracts/financial-templates/perpetual-multiparty/PerpetualLiquidatable.sol

```diff
@@ -241,7 +241,7 @@ contract PerpetualLiquidatable is PerpetualPositionManager {
         PositionData storage positionToLiquidate = _getPositionData(sponsor);
 
         tokensLiquidated = FixedPoint.min(maxTokensToLiquidate, positionToLiquidate.tokensOutstanding);
-        require(tokensLiquidated.isGreaterThan(0), "Liquidating 0 tokens");
```

Comment:
I needed to claw back some bytecode so I took some liberties with which require messages I removed. Please take a hard look at these and let me know what you think.

Reply:
this is sad but what choice do we have!

Reply:
I think these reverts whenever the user passes in 0 tokens don't need require messages because users are less likley to do this in the firs tplace